aws/adapter.go

```diff
@@ -64,7 +64,7 @@ const (
 
 	nameTag = "Name"
 
-	certificateARNTag = "ingress:certificate-arn"
+	certificateARNsTag = "ingress:certificate-arns"
```

Comment:
this will break existing installations, please make it upgradeable and set a deprecated flag